superduperdb/container/model.py

```diff
@@ -294,7 +294,9 @@ def predict(
             select = Serializable.deserialize(select)
 
         if db is not None:
+            logging.info(f'Adding model {self.identifier} to db')
             db.add(self)  # type: ignore[arg-type]
+            logging.info('Done.')
```

Comment:
Suggest either removing or adding more information ('Done' will mean very little in a log). I would probably err on the side of removing it, so that users don't feel we are overusing the logs.